components/CommentModal/CommentModal.js

```diff
@@ -0,0 +1,124 @@
+    const senatorId = profile && profile.senator ? profile.senator.id : null
+    const senatorEmail = senatorId ? GetMemberEmail(senatorId) : ""
+    const representativeEmail = representativeId ? GetMemberEmail(representativeId) : ""
+    const edit = useEditTestimony(user.uid, bill.BillNumber)
+    const positionMessage = "Select my support..(required)"
+    
+    const url = `mailto:${senatorEmail},${representativeEmail}?subject=My testimony on Bill ${bill ? bill.BillNumber : ""}&body=${testimony ? testimony.content : ""}`
+
+    const defaultPosition = testimony && testimony.position ? testimony.position : undefined
+  
+    const defaultAnonymous = testimony && testimony.anonymous ? testimony.anonymous : false
+    const defaultContent = testimony && testimony.content ? testimony.content : defaultTestimony
+    
+    const publishTestimony = async () => {
+      if (testimony.position == undefined || testimony.position == positionMessage) { return }
+      handleCloseTestimony()
+      await edit.saveDraft.execute(testimony)
+      await edit.publishTestimony.execute()
+    }
+    
+    return (
+     <Modal show={showTestimony} onHide={handleCloseTestimony} size="lg">
+        <Modal.Header closeButton onClick={handleCloseTestimony}>
+            <Modal.Title>{"Add Your Testimony" + (bill ? " for " + bill.BillNumber + " - " + bill.Title : "")}</Modal.Title>
+        </Modal.Header>
+
+        <Modal.Body>
+
+          <div className="container">
+            <div className="row">
+              <div className="col-sm align-middle">
+                <select className="form-control" defaultValue={defaultPosition} 
+                    onChange={e => {
+                        const newPosition = e.target.value
+                        if (newPosition) {
+                          const testimonyObject = {content: testimony.content, position: newPosition}
+                          setTestimony(testimonyObject)
+                        }
+                    }}>
+                    <option>{positionMessage}</option>
+                    <option value="endorse">Endorse</option>
+                    <option value="oppose">Oppose</option>
+                    <option value="neutral">Neutral</option>
+                </select>
+                <div className="form-check">
+                  <input className="form-check-input" type="checkbox" defaultValue={defaultAnonymous} id="flexCheckChecked"/>
+                  <label className="form-check-label" htmlFor="flexCheckChecked">
+                      Anonymous
+                  </label>
+                </div>
+                <div>
+                  <links.External href={url}>
+                      Send copy to your legislators
+                  </links.External>
+                </div>
+                <div>
+                  <links.External href={url}>
+                      Send copy to relevant committee
+                  </links.External>
+                </div>
```

Comment:
Should we show these if the user has not chosen legislators? Maybe a note to say "add your legislators in your profile to email them your testimony"

Reply:
I like it